framework/docs/source/ref-flower-network-communication.rst

```diff
@@ -62,7 +62,42 @@ Optional Network Connections
 Depending on the SuperLink and SuperNode configuration, Flower systems can have/use a
 number of additional network connections.
 
-Isolation mode
+Networking Interfaces
+~~~~~~~~~~~~~~~~~~~~~
+
+Each component — SuperLink, ``ServerApp``, SuperNode, and ``ClientApp`` — exposes ports
+for interacting with other Flower components. The SuperLink component includes three
```

Comment:
Technically not true. `ServerApp` and `ClientApp` don't have open ports. Also, while we often refer to `flwr-serverapp` with as just `ServerApp`, we should be more precise in reference docs. We should call it either `flwr-serverapp` or something like "the `ServerApp` process".